.github/workflows/main.yml

```diff
@@ -10,26 +10,25 @@ on:
         required: false
         type: boolean
         default: true
-    secrets:
-      REGISTRY_REDHAT_IO_USER:
-        required: true
-      REGISTRY_REDHAT_IO_PASSWORD:
-        required: true
 
 env:
   WF_CONFIG_REPO: parodos-dev/serverless-workflows-config
-
+  REGISTRY_REDHAT_IO_USER: ${{ secrets.REGISTRY_REDHAT_IO_USER }}
+  REGISTRY_REDHAT_IO_PASSWORD: ${{ secrets.REGISTRY_REDHAT_IO_PASSWORD }}
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+
     - name: Log in to Red Hat Registry
       uses: redhat-actions/podman-login@v1
       with:
         registry: registry.redhat.io
-        username: ${{ secrets.REGISTRY_REDHAT_IO_USER }}
-        password: ${{ secrets.REGISTRY_REDHAT_IO_PASSWORD }}
+        username: $REGISTRY_REDHAT_IO_USER
+        password: $secrets.REGISTRY_REDHAT_IO_PASSWORD
```

Comment:
why not use `REGISTRY_REDHAT_IO_PASSWORD`? or `${{ env.REGISTRY_REDHAT_IO_PASSWORD }}`?

Reply:
I tried 200 versions, based on different suggestions. nothing worked. all was reverted at the end.

Reply:
You tried that too I guess https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callsecrets
(putting the secret in the `workflow_call`)